packages/chats/src/index.ts

```diff
@@ -5,8 +5,10 @@ export * from './UIChatInput/ChatInputContainer';
 export * from './UIChatInput/useChatInputValue';
 export * from './BubbleQRCode';
 export * from './BubbleMedia';
+export * from './BubbleMedia/hooks';
 export * from './UILoadMoreButton';
 export * from './DuplicateImage';
+export * from './Lightbox/Lightbox';
```

Comment:
Can you add TODO:// about a desire to move it into independent `uikit.media` kit package?